### PR TITLE
Allow Movable Insulation to Support Equivalent Layer Glazing

### LINKED ISSUE #7385: SurfaceControl:MovableInsulation does not support equivalent layer window glazing but we have unit test for it
Issue overview
--------------
Noticed that the unit tests added for SurfaceControl:MovableInsulation in issue #5757 [HERE](https://github.com/NREL/EnergyPlus/blame/develop/tst/EnergyPlus/unit/HeatBalanceMovableInsulation.unit.cc#L97-L99) and [HERE](https://github.com/NREL/EnergyPlus/blame/develop/tst/EnergyPlus/unit/HeatBalanceMovableInsulation.unit.cc#L132-L134) and the pull request #5762 shows that Equivalent Layer Window glazing can be used as material layer. But testing shows that SurfaceControl:MovableInsulation does not support any of the equivalent layer window materials.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus v9.2.0
 - see the code [here](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/SurfaceGeometry.cc#L9713-L9773). The code requires that the material must have either thermal resistance or thermal conductivity and thickness to be valid in this object.

 Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (list location of defect file here)
 - [x] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by RKStrand**
Pull request overview
---------------------
Resolves #7385. A recent fix included a unit test that checked to see if movable insulation works with equivalent layer glass.  The unit test is successful and it appears from the code that this was considered an appropriate input for movable insulation.  Unfortunately, there was another place in the code that prevented that option.  This has now been fixed and the documentation has been clarified.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue #7385 

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add to input rules file for interfaces
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Required documentation updates?
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by Nigusse** [comment]
Tested the use of Glazing Equivalent Layer material with movable insulation object. It looks the model is working as expected. Below is a sample output, the outside wall surface temperature is impacted by the equivalent layer glazing used as "movable insulation layer" on the outside..

MovableInsulationImpact

File: doc/input-output-reference/src/overview/group-surface-construction-elements.tex [hunk 1]
```diff
@@ -3386,6 +3386,10 @@ \subsubsection{Inputs}\label{inputs-30-001}
 
 This value is the back side long-wave hemispherical emissivity of the glazing. The minimum value is 0.0, the maximum value is less than 1.0.~ The default value is 0.84.
 
+\paragraph{Field: Thermal Resistance}\label{field-thermal-resistance-equivglazing}
+
+This field is used to enter the thermal resistance (R-value) of the material layer. Units for this parameter are (m\(^{2}\)-K)/W. Thermal resistance must be greater than zero.  The default value is 0.158 which is roughly equivalent to a single layer of 1/4" glass.
```
Comment (Nigusse) [inline doc/input-output-reference/src/overview/group-surface-construction-elements.tex]: @RKStrand should we add a note that field is used with movable insulation only?

File: idd/Energy+.idd.in [hunk 2]
```diff
@@ -6960,13 +6960,20 @@ WindowMaterial:Glazing:EquivalentLayer,
...
        \default 0.84
-  N25; \field Back Side Infrared Emissivity
+  N25, \field Back Side Infrared Emissivity
        \note The back side long-wave hemispherical emissivity of the glazing.
        \type real
        \units dimensionless
        \minimum> 0.0
        \maximum< 1.0
        \default 0.84
+  N26; \field Thermal Resistance
```
Comment (Nigusse) [inline idd/Energy+.idd.in:6970]: @RKStrand should we add the same note here as well.

**Review by Nigusse — COMMENTED** [review]
@RKStrand if you address the two comments, this fix is ready to merge.

**Comment by RKStrand** [comment]
@Nigusse Thanks for your review, Bereket!  I have made changes per your suggestions and have committed those changes.  Assuming that the CI runs come back ok (which hopefully it will) and you are okay with these edits, this should be ready to merge.

**Comment by Nigusse** [comment]
I have reviewed this fix and approved for merging. @Myoldmopar would you like to look at it before I merge?

**Comment by Myoldmopar** [comment]
@Nigusse this looks good to me, feel free to merge.

**Pull request merged**